integreat_cms/cms/forms/custom_content_model_form.py

```diff
@@ -98,11 +94,10 @@ def clean(self) -> dict[str, Any]:
 
         return cleaned_data
 
-    # pylint: disable=too-many-branches, too-many-locals
     def clean_content(self) -> str:
         """
         Validate the content field (see :ref:`overriding-modelform-clean-method`) and applies changes
-        to ``<img>``- and ``<a>``-Tags to match the guidelines.
+        to ``<h1>``, ``<pre>``, ``<code>``, ``<img>``- and ``<a>``-Tags to match the guidelines.
```

Comment:
Nit: I think it would be easier to just say that it can update any element, otherwise we will always have to remember to keep this doc comment in sync with the implementation